utoipa-gen/src/path/property.rs

```diff
@@ -40,13 +40,14 @@ impl ToTokens for Property<'_> {
                 })
             }
 
-            if self.type_definition.is_array {
-                component.extend(quote! {
-                    .to_array_builder()
```

Comment:
Just wondering isn't this the same thing as the manual if below? The `.to_array_builder()` will ultimately do the same thing as `quote! {...}` does now. :thinking: